app/components/stories/complete/york/demographic-kpis/component.js

```diff
@@ -276,8 +276,8 @@ export default BaseRAGTile.extend(EditableFields, {
         return [
             {
                 name: 'displayed_kpi',
-                type: 'select',
-                contentPath: selectableKpis,
+                type: 'enum',
```

Comment:
Hey @SteAllan, there was already an editable field type for what we wanted. All you we needed was to change from `select` to `enum` then pass through a JSON string of an array for the select-2 content. (It will also want an `id` & `text` property on the array objects so select-2 knows how to handle it. Give it a test in the way it was failing before and see if it works :-)